mr/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -232,8 +269,10 @@ static void overlayTableProperties(Configuration configuration, TableDesc tableD
     map.put(InputFormatConfig.TABLE_SCHEMA, schemaJson);
 
     if (table instanceof Serializable) {
```

Comment:
Do we need this check?
I think it was introduced when MetaData tables were not `Serializable`. Now, all of the tables should work with `SerializableTable.copyOf(table)`, so I think we can remove this check

Reply:
No, we don't need this anymore. I've removed it, thanks!